internal/config/default/config.toml

```diff
@@ -24,6 +24,7 @@ limit = 0
   absorb = ["A"]
   split = ["s"]
   undo = ["u"]
+  redo = ["R"]
```

Comment:
`R` is already bound to _revert_. Normally, it should be bound to `ctrl+r` but that's also taken by _refresh_. `U` makes most sense given the remaining options.

Reply:
updated!